app/routes/models/claim-list.js

```diff
@@ -30,14 +19,14 @@ const getClaimTableHeader = (sortField) => {
       'data-url': 'claims/sort/type of visit'
     }
   },
-  {
+  (showSBI && {
```

Comment:
Where has this optional showing of SBI come from? I can't see that this existed before, and unclear why it would be something they'd want to hide.

Reply:
Previously the headers and rows for claims where rendered in a couple of places. The agreement page rendered them once (with SBI) using the viewModel, then rendered them again itself (without SBI) and passed both those and the viewModel version into the template for rendering. https://github.com/DEFRA/ffc-ahwr-backoffice/blob/main/app/routes/agreement.js#L105

The view then used the version without an SBI column, presumably because the SBI is already on the page within the application summary, and it would result in the same value for every row of that column.

<img width="1049" alt="Screenshot 2025-03-04 at 3 31 47 pm" src="https://github.com/user-attachments/assets/68374684-983a-4fce-8011-1e34c9192457" />

Reply:
Thanks. Good explanation. I had a feeling it was something along those lines, but it's just the sheer size of the changes and the limited time I have to read through them back and forth that stopped me from finding the exact answer.